tests/Model/Release/LibraryReleaseTest.php

```diff
@@ -65,4 +65,55 @@ public function testCountAllItems()
         $this->assertSame(3, $parent->countAllItems(false));
         $this->assertSame(4, $parent->countAllItems(true));
     }
+
+    public function testGetChildPriorVersions()
```

Comment:
I added this test. It's a bit unruly. Maybe we should break it up in smaller chunks.